src/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 SET(SRCS s3.cpp s3_http.cpp s3_io.cpp s3_signature_v2.cpp)
 SET(AWS_INSTALL_LIST)
 
-add_library(miniocpp SHARED ${SRCS})
+add_library(miniocpp STATIC ${SRCS})
```

Comment:
why is this being changed? @kkzi

Reply:
there are no export symbols in any header files, which cause link errors on windows with msvc.
therefore, it is more appropriate to use a static link library, for both gcc and msvc. 
define export symbols in header file is also a solution.
the most suitable solution is to support both dynamic linking and static linking, it will take some hours and need planning

Reply:
Understood, but this is a breaking change I don't think I can accept it right now. It is okay for this project to not build on windows. As long as it builds on Linux for now we are good.

Reply:
Actually chuck that, we are good here.